SimpleCalc/SimpleCalc/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+namespace SimpleCalc
+{
+    public enum operations { Multiplication = 1,
+                            Divide,
+                            Addition,
+                            Subtraction,
+                            Exponentiation };
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter first operand, please");
+
+            var input = Console.ReadLine();
+            var operand1 = int.Parse(input);
+            
+            Console.WriteLine(@"Select the arithmetic operation:
+1. Multiplication
+2. Divide
+3. Addition
+4. Subtraction
+5. Exponentiation");
+
+            input = Console.ReadLine();
+            var operation = (operations)int.Parse(input);
+
+            Console.WriteLine("Enter second operand, please");
+            input = Console.ReadLine();
+            var operand2 = int.Parse(input);
+
+            double result = 0;
+            string operationSymbol = "";
+
+            switch (operation)
+            {
+                case operations.Multiplication:
+                    result = operand1 * operand2;
+                    operationSymbol = " * ";
+                    break;
+
+                case operations.Divide:
+
+                    if (operand2 == 0)
+                    {
+                        Console.WriteLine("Divivding by zero is impossible");
+                        return;
+                    }
+
+                    result = operand1 / operand2;
+                    operationSymbol = " / ";
+                    break;
+
+                case operations.Addition:
+                    result = operand1 + operand2;
+                    operationSymbol = " + ";
+                    break;
+                case operations.Subtraction:
+                    result = operand1 - operand2;
+                    operationSymbol = " - ";
+                    break;
+                case operations.Exponentiation:
+                    operationSymbol = " ^ ";
+                    if (operand2 > 0)
```

Comment:
I think better way to implement exponentiation is to use Math.Pow(), but your implementation is ok too

Reply:
I agree with you. I don`t remember why I choose this way)